addons/headless/functions/fnc_transferGroups.sqf

```diff
@@ -89,8 +89,10 @@ private _numTransferredHC3 = 0;
                 _transfer = false;
             };
 
+            private _vehicle = vehicle _x;
+
             // No transfer if vehicle unit is in or crew in that vehicle is blacklisted
-            if (vehicle _x != _x && {(vehicle _x) getVariable [QXGVAR(blacklist), false]}) exitWith {
+            if (_vehicle != _x && {_vehicle getVariable [QXGVAR(blacklist), false] || {unitIsUAV _vehicle}}) exitWith {
```

Comment:
```suggestion
            if ((_vehicle getVariable [QXGVAR(blacklist), false]) || {unitIsUAV _vehicle}) exitWith {
```
always false for `objNull`

Reply:
Also, is this crew UAV check right?

Reply:
No, this checks if the unit's vehicle is a UAV. The UAV crew check happens here:
https://github.com/acemod/ACE3/blob/e183c34eae55d1090426fbaaf190fab7ca02aca1/addons/headless/functions/fnc_transferGroups.sqf#L83

It's the reason why I stuck with `vehicle` instead of `objectParent`, although we can easily change the check from `_vehicle != _x` to `!isNull _vehicle`.

---

The thing is, I'm not even sure if this is necessary. You should have UAV AI in a UAV, but I'm not sure if that's guaranteed.

Reply:
UAV can in fact have no crew (if created as empty vehicle through either createVehicle or Zeus)

Reply:
> UAV can in fact have no crew 

Not traditionally, true, but an UAV won't function without the special UAV AI (and you can't drag-drop one afterwards either, as the game/entity ain't smart enough to auto-convert to the UAV AI, last I tested in 2.10 or 2.12 or 2.14), which is needed by either
- hacking the UAV
- spawning the UAV with crew in the 1st place.